bootstrap/src/main/java/se/fortnox/reactivewizard/Main.java

```diff
@@ -41,27 +36,4 @@ protected void configure() {
             System.exit(-1);
         }
     }
-
-    /**
-     * Prepares a ConfigFactory before setting up Guice.
-     * <p>
-     * As logging can be part of the configuration file and the configuration file
-     * could be missing, we have a side effect of setting up and initializing a LoggingFactory
-     * that doesn't depend on ConfigFactory, if the configuration file is missing.
```

Comment:
It is not clear if this justification is still vald.

Reply:
it is no longer valid.